.github/workflows/auto_label_prs.yml

```diff
@@ -0,0 +1,23 @@
+name: Label PRs
+
+on:
+- pull_request
```

Comment:
I like that this only runs when a PR is `opened`, so we can change it afterwards if we don't like the labelling 👍

Reply:
Yes. Also I made it mandatory in my spec to disallow removing labels. Because, I could imagine seeing myself fighting a bot :p